src/main/java/frc/robot/commands/test/DriveVoltage.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.commands.test;
+
+import edu.wpi.first.networktables.GenericEntry;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+
+public class DriveVoltage extends CommandBase {
```

Comment:
I'm not sure this is necessary, it is put on the test tab but it doesn't seem to test anything, other than the fact that the motors would spin, which seems redundant with other tests.

Reply:
This and Steer volts are more legeacy test so we could remove them but rn they are still there

Reply:
I think then they should be removed. Same with the circle drive and other tests. If they don't have any purpose then they should be removed, and also the tests are all meant to test something, those commands do not do that right?